### ISSUE TITLE
SPIKE: Consider modifying local docker memory to match cloud.gov memory allocation

### ISSUE THREAD

**Issue by lbeaufort** [fecgov/fecfile-web-api#840]
In order to better replicate the resource limitations that are in cloud.gov,

Cloud.gov: 2GB, cpu MHz 3099.715

Consider modifying local docker memory to match cloud.gov app memory
 ```
 : + deploy:
 + resources:
 + limits:
 + memory: 2G
 ```

*   [x] Configure a local system setting in docker-compose.yml
*   [x] Run locust load testing to see effects reduced resources has locally.

### QA Notes ###
 
  
 null

### DEV Notes ###
 
  
 null

### Design ###
 
  
 null

**Comment by sasha-dresden** [comment]
To test this I ran locust with 150 users and 20 user/s ramp up speed. For a total of 5 minutes.
Results of unlimited memory

![Image](https://github.com/fecgov/fecfile-web-api/assets/149604256/75385e54-c915-497e-8c25-c73594d9d71c)

Results of limited memory:

![Image](https://github.com/fecgov/fecfile-web-api/assets/149604256/f60bceb7-c8df-49a7-8739-8ba7627371ba)


While the 2GB was marginally slower on overall response speed, it was a fairly small margin. However, where it did struggle was in requests per second, dropping from 54.9 to 30.3 which is a fairly significant margin. 

But otherwise, I don't see any reason why not to match our local docker configurations to the cloud.gov settings.  I also ran this for 500 users at 2 minutes to see how much load would effect matters and received almost identical results for both.

I know this wasn't mentioned in the ticket description but I wanted to call it out here. This was just modifying the memory. CPU is untouched. But unfortunately docker does not provide a simple way to restrict the CPU usage of a container. The only setting they have for that is restricting it based on a percentage of total CPU available, which  varies based on your local machine.
